src/SwitchField/SwitchField.js

```diff
@@ -31,7 +32,7 @@ export default function FormSwitch({
           component="div"
           color={disabled ? "textSecondary" : "textPrimary"}
         >
-          <Grid component="label" container alignItems="center" spacing={1}>
+          <Stack alignItems="center" component="label" direction="row">
             <Grid item>
```

Comment:
I think you forgot to remove all references to Grid. They don't provide any value now and just add extra items to the DOM.